### ISSUE TITLE
useLayoutEffect warnings when using Remix

### ISSUE THREAD

**Issue by tyteen4a03** [komarovalexander/ka-table#251]
I seem to be getting a bunch of warnings about useLayoutEffect when using this library with Remix:
```
Warning: useLayoutEffect does nothing on the server, because its effect cannot be encoded into the server renderer's output format. This will lead to a mismatch between the initial, non-hydrated UI and the intended UI. To avoid this, useLayoutEffect should only be used in components that render exclusively on the client. See https://reactjs.org/link/uselayouteffect-ssr for common fixes.
    at HeadCellContent (ka-table/Components/HeadCellContent/HeadCellContent.js:34:24)
    at div
    at div
    at th
    at HeadCell (ka-table/Components/HeadCell/HeadCell.js:34:33)
    at tr
    at HeadRow (ka-table/Components/HeadRow/HeadRow.js:23:36)
    at thead
    at exports.TableHead (ka-table/Components/TableHead/TableHead.js:32:36)
    at table
    at div
    at exports.TableWrapper (ka-table/Components/TableWrapper/TableWrapper.js:36:20)
    at div
    at exports.Table (ka-table/Components/Table/Table.js:35:33)
```

I worked around the issue by using `<ClientOnly>` from remix-utils but it would be great if this could also work SSR :D

**Comment by Lootvoet** [comment]
Any update on this?
Still have this issue when using NextJS with ka-table version 7.6.3.

**Comment by komarovalexander** [comment]
Hi @Lootvoet  thanks for the question, I expect release with the fix until the end of this week, I will keep you updated

**Comment by komarovalexander** [comment]
@Lootvoet @tyteen4a03 the issue has been fixed in 7.6.4
